Framework/PythonInterface/mantid/api/src/Exports/ExperimentInfo.cpp

```diff
@@ -42,7 +55,22 @@ void export_ExperimentInfo() {
       .def("getInstrument", &ExperimentInfo::getInstrument,
            return_value_policy<RemoveConstSharedPtr>(), args("self"),
            "Returns the :class:`~mantid.geometry.Instrument` for this run.")
-
+      .def("getResourceFilenames", &getResourceFilenames,
+           (arg("prefix"), arg("fileFormats"), arg("directoryNames"),
+            arg("date")),
+           "Compile a list of files in compliance with name pattern-matching,\n"
+           "file format, and date-stamp constraints\n\n"
+           "Ideally, the valid-from and valid-to of any valid file should\n"
+           "encapsulate the argument date. If this is not possible, then\n"
+           "the file with the most recent valid-from stamp is selected\n\n"
+           "prefix:         the name of a valid file must begin with this "
+           "pattern\n"
+           "fileFormats:    list of valid file extensions\n"
+           "directoryNames: list of directories to be searched\n"
+           "date :          valid-from and valid-to of a valid file "
```

Comment:
Nice documentation! Would it make sense to mention the format in which the `date` string should be given?

Reply:
https://github.com/mantidproject/mantid/pull/24798/commits/4c2177ad91268fbd7a9bf3a152861b65e78fe9db